python/lsst/validate/drp/plot.py

```diff
@@ -105,8 +100,8 @@ def plotAstrometry(mag, mmagerr, mmagrms, dist, match, good_mag_limit=19.5,
     ax[1].legend(loc='upper left')
     plotOutlinedLines(ax[1], dist_median, bright_dist_median)
 
-    plt.suptitle("Astrometry Check : %s" % plotbase.rstrip('_'), fontsize=30)
-    plotPath = plotbase+"check_astrometry.png"
+    plt.suptitle("Astrometry Check : %s" % plotBase.rstrip('_'), fontsize=30)
```

Comment:
are you expecting the path to include trailing underscores? It seems odd that the path is part of the plot title but maybe that's because `plotBase` is not documented.